tools/sst_dump_tool.cc

```diff
@@ -349,10 +353,12 @@ void print_help() {
     --file=<data_dir_OR_sst_file>
       Path to SST file or directory containing SST files
 
-    --command=check|scan|raw
+    --command=check|scan|raw|recompress
         check: Iterate over entries in files but dont print anything except if an error is encounterd (default command)
         scan: Iterate over entries in files and print them to screen
         raw: Dump all the table contents to <file_name>_dump.txt
+        recompress: recreates the SST file with different compression types and
```

Comment:
does it really rewrite the file with new compression setting, or does it only report the size? The description confuse me.